env/dev/env/config.cljs

```diff
@@ -1,6 +1,6 @@
 (ns env.config)
 
 (def figwheel-urls {:android "ws://localhost:3449/figwheel-ws",
- :ios "ws://192.168.0.9:3449/figwheel-ws",
+ :ios "ws://172.30.32.1:3449/figwheel-ws",
```

Comment:
could you please revert this file from PR, thanks

Reply:
Ugh, sorry,  and done in current commit.